draft-ietf-quic-transport.md

```diff
@@ -4584,13 +4584,14 @@ stateless_reset_token (0x0002):
   reset ({{stateless-reset}}) for the connection ID negotiated during the
   handshake.
 
-max_packet_size (0x0003):
+max_udp_size (0x0003):
 
-: The maximum packet size parameter is an integer value that limits the size of
-  packets that the endpoint is willing to receive.  This indicates that packets
-  larger than this limit will be dropped.  The default for this parameter is the
-  maximum permitted UDP payload of 65527.  Values below 1200 are invalid.  This
-  limit only applies to protected packets ({{packet-protected}}).
+: The maximum udp size parameter is an integer value that limits the size of
+  UDP payloads that the endpoint is willing to receive.  This indicates that
+  packets larger than this limit are likely not to be processed by the receiver.
+  The default for this parameter is the maximum permitted UDP payload of 65527.
+  Values below 1200 are invalid.  This limit only applies to protected packets
```

Comment:
What was the rationale for this text: `This limit only applies to protected packets ({{packet-protected}}).` ? Given the new clarification, I think we should just remove this sentence.

Reply:
I have no idea. Maybe @martinthomson or @janaiyengar knows?

I'm happy to remove it, if it's a mistake.

Reply:
See the issue for rationale.  However, with this change it can be removed.  It does create the issue I mentioned about Retry though.